artwork/ads/joinus.svg

```diff
@@ -624,4 +624,4 @@
          y="0"
          sodipodi:role="line"
          id="tspan456">Javascript | Java | Python | HTML | C++ | CSS</tspan></text>
-</g></svg>
```

Comment:
@abhayKashyap03 Unnecessary change. Remove this

Reply:
This was not a change I made, it must have happened accidentally.